data/systemd/snapd.service.in

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=Snap Daemon
-After=snapd.socket
+After=snapd.socket time-set.target
```

Comment:
Interestingly, I don't have any services in my system which would have a dependency on the time-set target, with the exception of the timers (as documented) and systemd-timesyncd.service.

Reply:
Fwiw, I'm trying this now with:
```
$ git diff
diff --git a/tests/nested/manual/minimal-smoke/task.yaml b/tests/nested/manual/minimal-smoke/task.yaml
index e4987af4c2..b6c9eed9b1 100644
--- a/tests/nested/manual/minimal-smoke/task.yaml
+++ b/tests/nested/manual/minimal-smoke/task.yaml
@@ -1,12 +1,13 @@
 summary: execute smoke tests in a nested Ubuntu Core VM that meets the minimal requirements
 
-systems: [ubuntu-16.04-64, ubuntu-18.04-64, ubuntu-20.04-64]
+systems: [ubuntu-16.04-64, ubuntu-18.04-64, ubuntu-20.04-64, ubuntu-22.04-64]
 
 environment:
     NESTED_ENABLE_SECURE_BOOT/secboot_disabled: false
     NESTED_ENABLE_TPM/secboot_disabled: false
     NESTED_ENABLE_SECURE_BOOT/secboot_enabled: true
     NESTED_ENABLE_TPM/secboot_enabled: true
+    NESTED_PARAM_RTC/disable_rtc: "-rtc base=1970-01-01,clock=vm"
 
 prepare: |
     tests.nested build-image core
```
to see if we can boot with a simulated non-rtc clock.